Global/Windows.gitignore

```diff
@@ -8,6 +8,7 @@ ehthumbs_vista.db
 
 # Folder config file
 Desktop.ini
+desktop.ini
```

Comment:
This can actually be folded up into the previous rule:

`[Dd]esktop.ini`

Reply:
Wow, I didn't know that. Thanks!